src/platform_impl/windows/monitor.rs

```diff
@@ -140,16 +131,16 @@ impl Window {
   }
 }
 
-pub(crate) fn get_monitor_info(hmonitor: HMONITOR) -> Result<winuser::MONITORINFOEXW, io::Error> {
-  let mut monitor_info: winuser::MONITORINFOEXW = unsafe { mem::zeroed() };
-  monitor_info.cbSize = mem::size_of::<winuser::MONITORINFOEXW>() as DWORD;
+pub(crate) fn get_monitor_info(hmonitor: HMONITOR) -> Result<MONITORINFOEXW, io::Error> {
+  let mut monitor_info = MONITORINFOEXW::default();
+  monitor_info.__AnonymousBase_winuser_L13558_C43.cbSize = mem::size_of::<MONITORINFOEXW>() as u32;
```

Comment:
Yeah, this one is a bit... scarier. It's disambiguating multiple struct/union fields in the same struct. It could require minor maintenance if the Windows SDK header changes, `win32metadata` regenerates from the new headers, and we update `windows-rs` to a version with the new name. But it is frozen as long as not all of those things happen, so it should not happen often. The compiler error in that case should even suggest the new name as something similar.

Reply:
I don't know when does win32metdata will generate these types, is it on every compile of `webview2-com-sys` or on every release of it?
I just don't want this to break builds unexpectedly for users of a certain tao version.

Reply:
These `Anonymous` members come from the `win32metadata` for the Windows Platform SDK, which is included in the `windows` crate and published with it for each version. The `webview2-com-sys` has a fixed version of the `windows` crate in its dependencies where it gets these types, and it mixes in the additional `win32metadata` for the WebView2 SDK. It should only change with an update to the `webview2-com-sys` which updated its own version of the `windows` crate, which had its own update to the Platform SDK `win32metadata`.

A re-build of TAO or Tauri won't do it, but theoretically it could require a code change when bumping the version of `webview2-com-sys`, if the Platform SDK itself changed and all of the upstream dependencies for `webview2-com-sys` had been updated in that version of `webview2-com-sys`. So at the point where someone was bumping the version of `webview2-com-sys` in TAO itself, they might run into a break they'd need to fix at the same time, but otherwise it should always be stable.

Reply:
cool, then this should be okay.